esptool.py

```diff
@@ -588,6 +569,192 @@ def load_section(self, section):
         return data
 
 
+class CesantaFlasher(object):
+    # This is "wrapped" stub_flasher.c, to  be loaded using run_stub.
+    FLASHER_STUB = """\
```

Comment:
Could the stub hex data be put at the very end of the file, please? I think that's the best compromise between readability and keeping everything in one source file.